packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -601,4 +602,12 @@ contract Attestations is
     return (attestation.status == AttestationStatus.Incomplete &&
       !isAttestationExpired(attestation.blockNumber));
   }
+
+  function isAttestationRequestSelectable(uint256 attestationRequestBlock)
+    internal
+    view
+    returns (bool)
+  {
+    return block.number < attestationRequestBlock.add(getRandom().randomnessBlockRetentionWindow());
```

Comment:
strictly also `block.number >= attestationRequestBlock + 4 && ` ? right?

Reply:
Strictly yes, but `4` should almost always be smaller